### ISSUE TITLE
v2.3.0 and what bout JRuby?

### ISSUE THREAD

**Issue by PetrKaleta** [rollbar/rollbar-gem#303]
In 2.3.0 you're using Oj gem. Ok, fine, and what about JRuby? You can't use globally gem with C extensions...

**Comment by PetrKaleta** [comment]
This is related pull request https://github.com/rollbar/rollbar-gem/pull/300

**Comment by PetrKaleta** [comment]
I'll suggest to use MultiJson, it's a good practice instead of being 3rd party JSON lib dependant.

**Comment by jondeandres** [comment]
@PetrKaleta we used MultiJson but we removed it since it was given us too many different behaviors from customers, some of them configuring it improperly. So we decided to just control ourself how to serialize the payload.

I think we didn't realize this wasn't failing in JRuby, sorry. This has been a misunderstanding with Travis cause we are really excluding JRuby builds there.

We'll fix this in next two days. Sorry for this.
